ST_mortality.c

```diff
@@ -1064,3 +1195,98 @@ void _kill_maxage(void) {
         }
     }
 }
+
+/**
+ * \brief Converts the biomass of cheatgrass to the % cover of cheatgrass.
+ * 
+ * This relationship between biomass and percent cover was derived by Maggie 
+ * England.
+ * 
+ * \param biomass is the biomass of cheatgrass.
+ * 
+ * \return A double between 0 and 1 representing the percent of the plot 
+ *         covered by cheatgrass.
+ * 
+ * \author Maggie England (derived the algorithm)
+ * \author Chandler Haukap (implemented the code)
+ * \date February 5 2020
+ * \ingroup MORTALITY_PRIVATE
+ */
+double _getCheatgrassCover(double biomass) {
+  int cover = biomass / (10.296 * Globals->plotsize);
+  return (cover > 1) ? 1 : cover;
```

Comment:
Again, I am confused with why we derive a value of 3 or 4 % cover and then set it to 1.

Reply:
@kpalmqui it looks like I made two mistakes here. First, I used an `int` instead of a `double`, then I misinterpreted the algorithm thinking that 1 = 100% and 0 = 0%. Commit 217f2c3 corrects both mistakes.

Reply:
@chaukap thanks - much improved! I will test and report back.